css/style.css

```diff
@@ -14,6 +8,197 @@ body {
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
+ 
+ * Module: HTML/CSS
+ * Lesson: 1
  */
 
+ @import url(https://fonts.googleapis.com/css?family=Share+Tech+Mono);
+
+ /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ /* Base reset */
+ :root {
+     --grey-dark: #545454;
+     --grey-light: #737373;
+     --white: #ffff;
+     --orange: #f15a29;
+     --orange-light: #f15a29;
+     --black: #00000;
+ }
+
+ * {
+     margin: 0px;
+     padding: 0px;
+ }
+
+ body {
+     -webkit-font-smoothing: antialiased;
+
+     margin: 0;
+     padding: 0;
+     border: 0;
+     font-size: 100%;
+     font: inherit;
+     vertical-align: baseline;
+     box-sizing: border-box;
+     font-family: "Roboto", sans-serif;
+     color: var(--grey-dark);
+ }
+
+ /*  Setion 1: navigation header  */
+ /* section 1 container */
+ .content1 {
```

Comment:
Could you try to choose a more specific name? Ask yourself, what does `.content1` do? Naming can be tricky... 😅  some class names can be content-semantic (e.g. `.header` or `.submenu`) and others visually semantic (e.g. `.column4` or `.btn-success`) or functional (e.g. what it does...) so I think in this case, `.content1` could be renamed `.flex-container`